rnd/autogpt_builder/src/components/CustomNode.tsx

```diff
@@ -402,8 +410,26 @@ const CustomNode: FC<NodeProps<CustomNodeData>> = ({ data, id }) => {
     return Object.values(newErrors).every((error) => error === null);
   };
 
+  const renderTooltip = (schema: any) => {
```

Comment:
* type?
* can we define this as a component so it can be referenced as `<SchemaTooltip schema={schema} />` rather than `{renderTooltip(schema)}`?

Reply:
fixed and component made